### DISCUSSION TITLE
[Feature Request]: HTML control type

### DISCUSSION THREAD

**Discussion by melaniebmn** [storybookjs/storybook#28421] [Ideas]
### Is your feature request related to a problem? Please describe

We started using Storybook for our web components design system and are running into an issue with the control types.
We have some components that use slots and allow for the input of HTML. From what we can tell (found on the documentation site / Stackoverflow), it is currently not possible to pass HTML into any control input type and have it rendered as HTML.

Being able to support the input of HTML into a control is vital for our project and a big blocker to not have available.

### Describe the solution you'd like

Provide a control type for HTML that allows the input of HTML and renders it as HTML.

### Are you able to assist to bring the feature to reality?

no

**Comment by juliannagreen1** [comment]
+1

**Comment by daaanny90** [comment]
I think it would be cool to see this happening. Any thoughts from Storybook team about this?
